api/src/main/java/ca/bc/gov/educ/api/batchgraduation/controller/JobLauncherController.java

```diff
@@ -253,7 +255,7 @@ public ResponseEntity<BatchJobResponse> launchRegGradSpecialJob(@RequestBody Stu
     }
 
     private void validateInput(BatchJobResponse response, StudentSearchRequest studentSearchRequest) {
-        if(studentSearchRequest.getPens().isEmpty() && studentSearchRequest.getDistricts().isEmpty() && studentSearchRequest.getSchoolCategoryCodes().isEmpty() && studentSearchRequest.getPrograms().isEmpty() && studentSearchRequest.getSchoolOfRecords().isEmpty()) {
+        if(studentSearchRequest.getStudentIDs().isEmpty() && studentSearchRequest.getPens().isEmpty() && studentSearchRequest.getDistricts().isEmpty() && studentSearchRequest.getSchoolCategoryCodes().isEmpty() && studentSearchRequest.getPrograms().isEmpty() && studentSearchRequest.getSchoolOfRecords().isEmpty()) {
```

Comment:
@infstar and @arybakov-cgi Checking .isEmpty() will cause a null pointer exception if the array is null. In order for this not to be dangerous, you will need to explicitly instantiate each property on the StudentSearchRequest object with = new ArrayList<>()

Reply:
@cditcher , the fix is part of GRAD2-2283 ticket. I have added empty arrays to StudentSearchRequest class

Reply:
@arybakov-cgi please just create a new PR from a branch called GRAD2-2817 with the fix. Because the bug was related to changes made in GRAD2-2817 we don't want the code mixed in with another PR that might take time to review and merge. Thanks.